ConfigGenerator/MainForm.cs

```diff
@@ -222,5 +222,43 @@ internal sealed partial class MainForm : Form {
 				Tutorial.OnAction(Tutorial.EPhase.BotNicknameFinished);
 			}
 		}
-	}
+
+        private void button_update_all_configs_Click(object sender, EventArgs e)
+        {
+            GlobalConfig globalConfig = GlobalConfig.Load(Path.Combine(SharedInfo.ConfigDirectory, SharedInfo.GlobalConfigFileName));
+            if (!string.IsNullOrEmpty(globalConfig.CurrentCulture))
+            {
+                try
+                {
+                    // GetCultureInfo() would be better but we can't use it for specifying neutral cultures such as "en"
+                    CultureInfo culture = CultureInfo.CreateSpecificCulture(globalConfig.CurrentCulture);
+                    CultureInfo.DefaultThreadCurrentCulture = CultureInfo.DefaultThreadCurrentUICulture = culture;
+                }
+                catch (CultureNotFoundException)
+                {
+                    Logging.LogGenericErrorWithoutStacktrace(CGStrings.ErrorInvalidCurrentCulture);
+                }
+            }
+
+            ConfigPage globalCfgPage = new ConfigPage(globalConfig);
```

Comment:
You can't create new ConfigPage when user clicks button - you should use ones that are already loaded, not create new ones.